src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;
+  padding: 1rem;
+  max-width: 20rem;
+  flex-grow: 1;
+  row-gap: .7rem;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  border-bottom: 1px solid;
+
+  & img {
+    position: relative;
+    width: 100%;
+    border-radius: 1rem;
+  }
+
+  .image-container {
+    position: relative;
+    height: 10rem;
+    background-image: url("../../assets/britnie.jpg");

```

Comment:
Omdat we de afbeeldingen moeten kunnen doorgeven vanuit een potentiële backend, is het minder handig om background-images te gebruiken. In dit geval volstaat een 'gewone' image ook! Je hebt nog steeds een image-container nodig met een border-radius, maar deze heeft een extra property `overflow: hidden` nodig zodat de `<img>` die je daarin zet, zich netjes houdt aan de ronde hoekjes :)